File: C-Python_Solving_50_Problems/14-Swap_Two_Numbers.py
# Python program to swap two numbers
# take inputs
num1 = float(input('Enter first number: '))
num2 = float(input('Enter second number: '))

# print original numbers
print('Original numbers: {0} and {1}'.format(num1, num2))

# swap numbers
temp = num1
num1 = num2
num2 = temp

# print swapped numbers
print('Swapped numbers: {0} and {1}'.format(num1, num2))

File: C-Python_Solving_50_Problems/14-Swap_Two_Numbers.c
#include <stdio.h>

int main() {
    float num1, num2, temp;

    // take inputs
    printf("Enter first number: ");
    scanf("%f", &num1);
    printf("Enter second number: ");
    scanf("%f", &num2);

    // print original numbers
    printf("Original numbers: %.2f and %.2f\n", num1, num2);

    // swap numbers
    temp = num1;
    num1 = num2;
    num2 = temp;

    // print swapped numbers
    printf("Swapped numbers: %.2f and %.2f\n", num1, num2);

    return 0;
}
